var/spack/repos/builtin/packages/py-shapely/package.py

```diff
@@ -35,22 +36,29 @@ class PyShapely(PythonPackage):
     version("1.8.0", sha256="f5307ee14ba4199f8bbcf6532ca33064661c1433960c432c84f0daa73b47ef9c")
     version("1.7.1", sha256="1641724c1055459a7e2b8bbe47ba25bdc89554582e62aec23cb3f3ca25f9b129")
     version("1.7.0", sha256="e21a9fe1a416463ff11ae037766fe410526c95700b9e545372475d2361cc951e")
-    version("1.6.4", sha256="b10bc4199cfefcf1c0e5d932eac89369550320ca4bdf40559328d85f1ca4f655")
+    version(
+        "1.6.4",
+        sha256="b10bc4199cfefcf1c0e5d932eac89369550320ca4bdf40559328d85f1ca4f655",
+        deprecated=True,
+    )
 
-    depends_on("c", type="build")  # generated
+    depends_on("c", type="build")
 
     # pyproject.toml
-    depends_on("py-cython", when="@2.0.2:", type="build")
-    depends_on("py-cython@0.29:0", when="@2.0.0:2.0.1", type="build")
-    depends_on("py-cython@0.29.24:2", when="@:1", type="build")
-    depends_on("py-setuptools@61:", when="@2:", type="build")
-    depends_on("py-setuptools@:63", when="@:1", type="build")
-    depends_on("py-numpy@1.14:2", when="@2.0.4:", type=("build", "link", "run"))
-    # https://github.com/shapely/shapely/issues/1972
-    depends_on("py-numpy@1.14:1", when="@2.0.0:2.0.3", type=("build", "link", "run"))
-    depends_on("py-numpy@:1", when="@1", type=("build", "link", "run"))
-    depends_on("py-pytest", type="test")
-    depends_on("py-pytest-cov", type="test")
+    with default_args(type="build"):
+        depends_on("py-cython", when="@2.0.2:")
+        depends_on("py-cython@0.29:0", when="@2.0.0:2.0.1")
+        depends_on("py-cython@0.29.24:2", when="@:1")
+        depends_on("py-setuptools@61:", when="@2:")
+        depends_on("py-setuptools@:63", when="@:1")
+
+    with default_args(type=("build", "link", "run")):
+        depends_on("py-numpy@1.14:2", when="@2.0.6:")
```

Comment:
🤔 According to `pyproject.toml` this should be a newer minimum:

```
...
    "oldest-supported-numpy; python_version<'3.9'",
    "numpy>=1.25; python_version>='3.9'",
    "setuptools>=61.0.0",
...
```

BUT I see `requires.txt` does match.  Which should it be?